local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
+# header note.
+#Wed Sep 07 10:00:38 EDT 2022
+sdk.dir=C\:\\Users\\mamin\\AppData\\Local\\Android\\Sdk
```

Comment:
Noting this for everyone that we will have to address this file each after this merges in. Don't worry, no problems. Once my PR #21 gets merged in this file will be removed from the repo and exist only locally for us all. With that being that case. We should merge this PR, then PR #21, then PR #22 .